src/sentry/static/sentry/app/views/organizationDiscover/resultManager.jsx

```diff
@@ -66,13 +101,23 @@ export default function createResultManager(queryBuilder) {
    * @returns {Void}
    */
   function reset() {
-    for (let key in data) {
-      data[key] = {query: null, data: null};
-    }
+    data = getDefault();
+  }
+
+  /**
+   * Resets all data
+   *
+   * @returns {{baseQuery: {query: null, data: null, next: null, previous: null}, byDayQuery: {query: null, data: null}}}
```

Comment:
nit: `@returns {Object}` would be fine here, you don't need to print the entire object again